backend/src/main/resources/db/migration/V000__create_initial_tables.sql

```diff
@@ -0,0 +1,91 @@
+CREATE TYPE EXCHANGE_REQUEST_STATUS AS ENUM ('PENDING', 'ACCEPTED', 'REJECTED', 'CANCELLED');
+CREATE TYPE CREDIT_CARD_TYPE AS ENUM ('VISA', 'MASTERCARD');
+CREATE TYPE FIELD_POSITION AS ENUM ('GOALKEEPER',
+                                    'MIDFIELDER',
+                                    'DEFENDER',
+                                    'FORWARD');
+CREATE TYPE EXCHANGE_OFFER_STATUS AS ENUM ('PENDING', 'ACCEPTED', 'REJECTED', 'COUNTEROFFER', 'CANCELLED');
+CREATE CAST (VARCHAR AS FIELD_POSITION) WITH INOUT AS IMPLICIT;
+CREATE CAST (VARCHAR AS EXCHANGE_REQUEST_STATUS) WITH INOUT AS IMPLICIT;
+CREATE CAST (VARCHAR AS EXCHANGE_OFFER_STATUS) WITH INOUT AS IMPLICIT;
+CREATE CAST (VARCHAR AS CREDIT_CARD_TYPE) WITH INOUT AS IMPLICIT;
+
+CREATE TABLE IF NOT EXISTS users (
+    id BIGSERIAL PRIMARY KEY,
+    first_name VARCHAR(100) NOT NULL,
+    last_name VARCHAR(100) NOT NULL,
+    email VARCHAR(100) NOT NULL,
+    birthday DATE NOT NULL,
+    username VARCHAR(20) NOT NULL UNIQUE,
+    auth0_sub VARCHAR(100) UNIQUE --Identificador de usuario de Auth0
+);
+
+CREATE TABLE IF NOT EXISTS card(
+    id BIGSERIAL PRIMARY KEY,
+    name VARCHAR(50) NOT NULL,
+    player_position FIELD_POSITION NOT NULL,
+    player_number SMALLINT NOT NULL,
+    country VARCHAR(100) NOT NULL,
+    photo_url VARCHAR(255)
+);
+
+--Consultar con Jonathan, aparentemente almacenar tarjetas de crédito no es buena práctica
```

Comment:
Usually credit cards are stored by third parties. It's not common to store them on our applications. And if we do we cipher the values and encrypt the database